packages/fdc3-get-agent/src/util/Picocolors.ts

```diff
@@ -0,0 +1,114 @@
+/* 
+From picocolors (imported due to issues building in both ESM and CommonJS) 
+https://github.com/alexeyraspopov/picocolors
+
+ISC License
```

Comment:
I'm a bit worried about this license.  A lot of FDC3's users have automated tools for license scanning and approvals.   If FDC3 suddenly ends up being mixed-license (Apache + ISC) then it may get flagged.  

According to opensource.org https://opensource.org/license/isc-license-txt. this license is currently not categorised as open source.  Is there an alternative with an Apache license we can use?

Reply:
I intend to remove it and replace with a custom implementation - this was a result of my frustration at not finding a lib for this that would build under both es6 and commonjs. I was going to do that in the debug logs PR...

Reply:
Actually, it is an OpenSource license, approved by the Open Source Initiative - the categorizations they display are within the set of Open Source licenses and are:
![image](https://github.com/user-attachments/assets/1cfe0ac7-8462-4530-9bd5-46b129484707)
Its also the default license on NPM packages created with `npm init` (I'm told).

Regardless I intend to replace this with a cusotm implementation.

Reply:
ok great. Yes, I misunderstood that.

Reply:
picocolors removed / replaced